src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -42,6 +42,36 @@ THE SOFTWARE.
                 <!-- Similar to f:dropdownDescriptorSelector, but adds fallback content to block, and JENKINS-25130 adds per-selection help: -->
                 <j:set var="item" value="${it.getItem(request)}"/>
                 <d:taglib uri="local">
+                    <!-- this tag wraps the dropdown list in a div but only in div-based forms
+                        using a div in table-based forms will crash the layout generation -->
+                    <d:tag name="snippetizerOptionsWrapper">
+                       <j:choose>
+                           <j:when test="${divBasedFormLayout}">
+                               <style>
+                                   .snippetizer-options-wrapper .setting-input.dropdownList {
+                                       max-width: calc(100% - 2rem);
+                                   }
+
+                                   .snippetizer-help {
+                                       position: relative;
+                                   }
+
+                                   .snippetizer-help a.help-button {
+                                       position: absolute;
+                                       top: -2rem;
+                                   }
+                               </style>
+
+                               <div class="snippetizer-options-wrapper">
+                                   <d:invokeBody />
+                               </div>
```

Comment:
@fqueiruga  
Could we include this div regardless of whether we're using div-based layout?  Or would that explode?

Reply:
it will explode, that's why i created this tag that noops on table layouts.